CHANGELOG.md

```diff
@@ -56,8 +56,11 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 Here is an overview of all new **experimental** features:
 
+
 - **General**: TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 
+- **Prometheus Metrics**: Introduce number of paused ScaledObjects in Prometheus metrics ([#4430](https://github.com/kedacore/keda/issues/4430))
```

Comment:
Could you move this to new section?

Reply:
new section where?

Reply:
Could you please explain more.
I don't get

Reply:
![image](https://github.com/kedacore/keda/assets/36899226/fcbe1881-ebac-4440-9d7e-28590a19040c)

Reply:
Oh alright I will do that

Reply:
Done